### PR TITLE
Add instructions for how to run custom Docker images

### PR DESCRIPTION
**Pull request by schen2166**
### Additional Instructions For Custom Docker Images

Added description for how to run custom docker image after building them
Made some additions to help explain/clarify the `docker build` command

### CODE REVIEW

**Comment by kylelaker** [comment]
Please work with @tuckerzp to remove the unrelated changes from this branch.

**Comment by kylelaker** [comment]
I didn't notice this before but I think there's something a little wonky with your git config was well. Please ensure `user.name` and `user.email` are configured correctly (for this repository and globally). @tuckerzp should be able to support with that also.

File: all-in-one/BUILDING-TESTING.md [hunk 1]
```diff
@@ -24,8 +24,17 @@ files may be passed to the `VIEWER_PATH` and `REST_PATH` build args as described
 [viewer-artifact]: https://github.com/EasyDynamics/oscal-react-library/releases/latest/download/oscal-viewer.zip
 [package]: https://github.com/EasyDynamics/oscal-rest-service/packages/1369238
 [pat]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
-
+[all-in-one]: https://github.com/EasyDynamics/oscal-editor-deployment/tree/main/all-in-one
+[docker-build-doc]: https://docs.docker.com/engine/reference/commandline/build/
```
Comment (Bronstrom) [inline all-in-one/BUILDING-TESTING.md:28]:
```suggestion
[docker-build-doc]: https://docs.docker.com/engine/reference/commandline/build

```

File: all-in-one/BUILDING-TESTING.md [hunk 2]
```diff
@@ -24,8 +24,17 @@ files may be passed to the `VIEWER_PATH` and `REST_PATH` build args as described
...
 ### Creating the Image
+The skeleton of the `docker build` is as follows the following syntax
+`docker build [OPTIONS] PATH | URL | -`
+Documentation for the docker build command can be found [here][docker-build-doc]
```
Comment (Bronstrom) [inline all-in-one/BUILDING-TESTING.md:32]:
```suggestion
Additional documentation for the docker build command can be found [here][docker-build-doc].
```

File: all-in-one/BUILDING-TESTING.md [hunk 3]
```diff
@@ -24,8 +24,17 @@ files may be passed to the `VIEWER_PATH` and `REST_PATH` build args as described
...
+
+Note: The following `OPTION` tags are used below
+`--tag` is the `OPTION` that names the docker image you are creating
+`--build-arg` is the `OPTION` that specifies how the image is being built
```
Comment (Bronstrom) [inline all-in-one/BUILDING-TESTING.md:36]:
```suggestion
`--build-arg` is the `OPTION` that specifies how the image is being built.
```

File: all-in-one/BUILDING-TESTING.md [hunk 4]
```diff
@@ -24,8 +24,17 @@ files may be passed to the `VIEWER_PATH` and `REST_PATH` build args as described
 [viewer-artifact]: https://github.com/EasyDynamics/oscal-react-library/releases/latest/download/oscal-viewer.zip
 [package]: https://github.com/EasyDynamics/oscal-rest-service/packages/1369238
 [pat]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
-
+[all-in-one]: https://github.com/EasyDynamics/oscal-editor-deployment/tree/main/all-in-one
+[docker-build-doc]: https://docs.docker.com/engine/reference/commandline/build/
 ### Creating the Image
+The skeleton of the `docker build` is as follows the following syntax
+`docker build [OPTIONS] PATH | URL | -`
```
Comment (Bronstrom) [inline all-in-one/BUILDING-TESTING.md:31]:
```suggestion
`docker build [OPTIONS] PATH | URL | -`.
```

File: all-in-one/BUILDING-TESTING.md [hunk 5]
```diff
@@ -64,5 +73,10 @@ most cases this is not necessary as the `_GIT_BRANCH` build arguments support bo
 ## Testing
 
 The container can be tested using end-to-end Cypress tests.
-
 More information on testing can be found in [end-to-end-tests.](../end-to-end-tests)
+
+After you successfully create a new docker image, to pull the newly created image run
```
Comment (Bronstrom) [inline all-in-one/BUILDING-TESTING.md:78]:
```suggestion
After you successfully create a new docker image, to pull the newly created image run:

```

**Review by Bronstrom — CHANGES_REQUESTED** [review]
Looking good so far. I have a couple suggestions to make the syntax consistent among the file.

**Comment by Bronstrom** [comment]
Not sure if anyone else tried the method of downloading OSCAL Viewer and the REST service with `packages_pull.sh` script, but it appears the names don't follow what's noted in the documentation. The `oscal-viewer` is `viewer` and `oscal-rest.jar` is `oscal-rest-service.jar`. I believe this should be noted in the documentation somewhere or the names be updated in the script.

**Comment by kylelaker** [comment]
> the names be updated in the script

This is personally my preferred solution

**Comment by Bronstrom** [comment]
@kylelaker I think that is the best solution too. I quickly put together a PR to address this: #107

**Review by Bronstrom — APPROVED** [review]
No problem! Looks good to me

File: all-in-one/BUILDING-TESTING.md [hunk 6]
```diff
@@ -63,6 +72,11 @@ most cases this is not necessary as the `_GIT_BRANCH` build arguments support bo
 
 ## Testing
 
-The container can be tested using end-to-end Cypress tests.
-
+The container can be tested using end-to-end Cypress tests:
 More information on testing can be found in [end-to-end-tests.](../end-to-end-tests)
+
+After you successfully create a new docker image, to pull the newly created image run:
```
Comment (mikeisen1) [inline all-in-one/BUILDING-TESTING.md:78]:
```suggestion

After you successfully create a new docker image, to run the newly created image execute:
```

**Review by mikeisen1 — CHANGES_REQUESTED** [review]

File: all-in-one/BUILDING-TESTING.md (hunk 6 above)
Comment (schen2166) [reply all-in-one/BUILDING-TESTING.md]: I like this suggestion, small change that phrases the sentence better

**Comment by schen2166** [comment]
Tests are very likely failing because the catalog is not loaded in time.

**Review by mikeisen1 — APPROVED** [review]

**Pull request merged**